plugins/tiddlywiki/menubar/language/zh-Hans/Translations.multids

```diff
@@ -0,0 +1,25 @@
+title: $:/plugins/tiddlywiki/menubar/language/zh-Hans/
```

Comment:
@linonetwo .. It seems the translations are part of the plugin here. That's not what users want. I personally do _not_ want to carry 32 translations with every core plugin, since I do only need 2 or 3 at most. 

As a user I do expect them to be added individually

Reply:
This is not that big. If this is not very necessary, I can leave it to next PR. I may be busy on https://github.com/tiddly-gittly/TidGi-Mobile these days.

Reply:
@pmario while I understand your position on this, I do not think that it is the way that we should go.

The architecture developed here will work with translations that are integral or are packaged as a separate plugin. For the core plugins, I do not think it is acceptable to expose a separate plugin for each translation of the menu bar. I think there are two viable options:

* Place the translations for the core plugins in the associated language plugin. So the fr-FR plugin would contain the French core translations, and those for the menu bar plugin etc.
* Place the translations for the core plugins within the plugin itself

The key point here is that the usability of keeping translations together absolutely trumps any phantom concerns about saving a few hundred bytes.

Reply:
>The key point here is that the usability of keeping translations together absolutely trumps any phantom concerns about saving a few hundred bytes.

It's not a view hundred bytes. 

menubar has about 1400Byte of text x 32 languages atm is about 45kByte of language related text. 
The plugin itself is exported as JSON has 25kByte. So the plugin-size will be almost tripled. 
Since the goal may be to have more translations, it will be worse.

Reply:
@pmario I think Jeremy means if it is big enough, we can still move them out, but not placed in sub-plugin -- place them in core language plugin instead -- for core plugins. I can try this later.